index.html

```diff
@@ -823,6 +823,375 @@ <h2>Evidence</h2>
+}
+        </pre>
+
+
+   </section>
+
+
+   <section>
+        <h2>Subject Delegates to Holder</h2>
+
+     <p>
+This is supported in the data model in two different ways: through the use of either recursion or profiles. Both methods involve the subject issuing a new credential to the delegated holder.
+    </p>
+
+    <p>
+This process of recursion could continue without restriction. Therefore 
+a new property “delegatable”, is added to the data model.
+
+     </p>
+
+   <section>
+
+   <h3> Delegatable Property </h3>
+
+  <p>
+The delegatable property, a Boolean, states whether a credential is 
+delegatable or not. If it is present with the value true, or not present, 
+then the credential is delegatable to a third party. If it is present with 
+the value false then it is not delegatable.
+
+  </p>
+
+        <pre class="example nohighlight" title="An example of a credential that is not 
+delegatable">
+{
+  "id": "http://dmv.example.gov/credentials/3732",
+  "type": ["Credential", "ProofOfAgeCredential"],
+  "issuer": "https://dmv.example.gov/issuers/14",
+  "issued": "2010-01-01",
+  "delegatable": false,
+  "claim": {
+    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    "ageOver": 21,
+  },
+  "proof": { ... }
+}
+        </pre>
+
+       </section>
+       
+       <section>
+
+     <h3> Delegation through Recursion </h3>
+
+    <p>
+With recursion, the subject issues a credential to the delegated holder in which: 
+ the issuer is the subject, the claim contains 
+the id of the delegated holder, and the claim property is the credential that 
+was originally issued to the subject. An example is provided below.
+    </p>
+
+ <pre class="example nohighlight" title="An example of a credential delegated 
+from a subject to a holder that is valid for 2 days">
+
+{ 
+  "id": "1234",
+  "type": ["Credential", "EmbeddedDelegatedCredential"],
+  "issuer": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+  "issued": "2010-01-06T00:00:00Z ",
+  "expires": "2010-01-08T00:00:00Z ",
+  "claim": {
+    "id": "did:example:ebfeb1f712ebc6f1c2123456789",
+    "credential”: {
+       "id": "http://example.gov/credentials/3732",
+       "type": ["Credential", "Role"],
+       "issuer": "https://dmv.example.gov",
+       "issued": "2010-01-01",
+       "claim": {
+         "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+         "role": "Project Manager”
+         },
+       "revocation": {
+       "id": "http://example.gov/revocations/738",
+       "type": "SimpleRevocationList2017" },
+       "proof": {
+       "type": "LinkedDataSignature2015",
+       "created": "2016-06-18T21:19:10Z",
+       "creator": "https://example.com/jdoe/keys/1",
+       "domain": "json-ld.org",
+       "nonce": "598c63d6",
+       "signatureValue": "BavEll0/I1zpYw8XNi1bgVg/sCneO4Jugez8RwDg/
++MCRVpjOboDoe4SxxKjkCOvKiCHGDvc4krqi6Z1n0UfqzxGfmatCuFibcC1wps 
+PRdW+gGsutPTLzvueMWmFhwYmfIFpbBu95t501+rSLHIEuujM/+PXr9Cky6Ed +W3JT24="
+    }
+  }
+},
+"proof": { ... } }
```

Comment:
fix indents from "proof" above down to here.

Reply:
proof index fixed